MakerTracker/DBModels/MakerTrackerContext.cs

```diff
@@ -23,6 +23,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
 
         public DbSet<MakerTracker.DBModels.MakerOrder> MakerOrder { get; set; }
+
+
+        public DbSet<MakerTracker.DBModels.Transaction> Transaction { get; set; }
```

Comment:
I would pluralize this since it's a set of Transactions... I also realize I didn't do that with MakerOrder right about it :)